frontend/src/lib.rs

```diff
@@ -30,6 +33,7 @@ pub struct Context {
     pub chains: HashMap<i32, model::Chain>,
     pub tags: HashMap<i32, model::Tag>,
     pub services: HashMap<i32, model::Service>,
+    pub lists: HashMap<Uuid, model::StoredList>,
```

Comment:
Using UUID as it's an easy unique key. I assume you used `i32` for database autoincrements, not sure if we want to continue with that here, since this data never touches the DB.